ui/apps/platform/src/Containers/NetworkGraph/TopologyComponent.tsx

```diff
@@ -90,18 +98,26 @@ const TopologyComponent = ({
     const { hasReadAccess } = usePermissions();
     const hasReadAccessForNetworkPolicy = hasReadAccess('NetworkPolicy');
 
+    const { detailID: selectedExternalIP } = useParams();
+
     const firstRenderRef = useRef(true);
     const history = useHistory();
     const controller = useVisualizationController();
     const [defaultDeploymentTab, setDefaultDeploymentTab] = useState(deploymentTabs.DETAILS);
 
     const closeSidebar = useCallback(() => {
-        const queryString = clearSimulationQuery(history.location.search);
+        let queryString = clearSimulationQuery(history.location.search);
+        queryString = clearSearchQueryValue(queryString, 'sidePanel');
         history.push(`${networkBasePath}${queryString}`);
     }, [history]);
 
-    function onNodeClick(ids: string[]) {
-        const newSelectedId = ids?.[0] || '';
+    type OnNavigateArgs = {
+        nodeID: string;
+        externalIP?: string;
+    };
+
+    function onNavigate({ nodeID, externalIP }: OnNavigateArgs) {
```

Comment:
Changing the name to `onNavigate` because it makes better sense now that we're dealing with selecting External IPs